crates/neon/src/types_impl/mod.rs

```diff
@@ -1158,6 +1120,7 @@ impl JsFunction {
     /// Calls this function.
     ///
     /// **See also:** [`JsFunction::call_with`].
+    #[deprecated(since = "TBD", note = "use `JsFunction::bind` instead")]
```

Comment:
I don't think this one should be deprecated. It's the low level function that doesn't have the overhead of casting and deserialization. I think we should keep it.